libnmstate/state.py

```diff
@@ -51,7 +52,58 @@
 
 
 @total_ordering
-class RouteEntry(object):
+class StateEntry:
+    def __init__(self):
+        pass
+
+    def keys(self):
```

Comment:
Why is it now public?

Reply:
If private, child class cannot override it.

Reply:
Sure it can... children "see" their parents privates.
As I see it, it is for external usage. When we do `self._*`, it is internal.

I do not really mind exposing `keys` outside, but then it better follow py3 convention and make it a view/iterator.

Reply:
The change should have been done on the previous commit.

Reply:
You may check (https://paste.fedoraproject.org/paste/KlymFdt9VJjBwEB5UxkMjA) to know why `__keys()` cannot works.